test/fuzz/transformation_replace_id_with_synonym_test.cpp

```diff
@@ -1851,7 +1851,9 @@ TEST(TransformationReplaceIdWithSynonymTest,
         %101 = OpCopyObject %17 %20 ; A non-constant version of %20
          %14 = OpAccessChain %13 %11 %12
          %21 = OpAtomicLoad %6 %14 %15 %20
+         %22 = OpAtomicExchange %6 %14 %15 %20 %16
                OpStore %8 %21
+               OpAtomicStore %14 %15 %20 %12
```

Comment:
Thanks for improving the test to feature AtomicExchange and AtomicStore.

It would be even better if you could extend it further to feature the other atomic operations that are available with the Shader capability, so that we have good coverage.

Reply:
Done.